alchemy-web/docs/providers/vercel/project-domain.md

```diff
@@ -0,0 +1,44 @@
+# ProjectDomain
+
+Add and manage domains for Vercel projects.
+
+## Authentication
+
+To use this resource, you must authenticate with Vercel. You can do this in one of two ways:
+
+1. **Environment Variable:** Set `VERCEL_ACCESS_TOKEN` in your `.env` file.
+2. **Direct Prop:** Pass `accessToken` directly in your resource configuration.
+
+## Examples
+
+### Minimal
+
+```ts
+const domain = await ProjectDomain("my-app.com", {
+  name: "my-app.com",
+  project: "prj_123",
+});
+```
+
+### With `accessToken`
+
+
+```ts
```

Comment:
nit: extra new line. format doc

Reply:
Resolved via 077b9774f8d75a3b8326d3923af67b96dd128141

Turns out the issue was `formatOnSave` wasn't enabled – 9485cc07132c724a287243c029659124c1ae21ab